cn/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -728,7 +728,7 @@ spec:
               name: jmx
             - containerPort: 9042
               name: cql
-              # If you need it it is going away in C* 4.0
+              # If you need it is going away in C* 4.0
```

Comment:
This isn't quite the right change, although you're correct that the original doesn't read properly. Try this:
"If you need it, it will go away in C* 4.0."
This whole comment and the larger doc could use more fixing, but this much will do for now.

Reply:
thanks~